### DISCUSSION TITLE
[Q&A] Created a new plugin, what's next?

### DISCUSSION THREAD

**Discussion by zbalkan** [fluent/fluentd#4526] [Q&A]
### What is a problem?

I used the pan anonymizer package but it was not properly helping my use case as is. Therefore, I improved it a little bit by allowing regex patterns to make masking configurable.

I have not published as a gem, and updated the instructions to use the Github repo as the source.

What do I need to get it published on fluentd page so that people who may need this functionality could find it?

Edit: Here's the link to the repo: https://github.com/zbalkan/fluent-plugin-pan-anonymizer

### Environment

```markdown
- Fluentd version: n/a
- TD Agent version: n/a
- Fluent Package version: n/a 
- Docker image (tag): n/a
- Operating system: n/a
- Kernel version: n/a
```

**Comment by daipom** [comment]
You sent the PR long ago, but there is no response.

https://github.com/kanmu/fluent-plugin-pan-anonymizer/pull/8

The most preferable way is to reflect the improvements to the original repo.
However, the last update was 6 years ago, so unfortunately it may no longer be maintained.

Let's mention the person who was developing this plugin.
If there is no response after waiting for a while, let's consider releasing the forked one as a new gem.
There is no problem if you follow this license:

https://www.apache.org/licenses/LICENSE-2.0

Basically, you should specify the original author's information in the README.

**Reply by zbalkan** [reply]
Yes, that was my intention. I'd like to rename the repo, maybe just "fluent-plugin-pan-anonymizer-2" with a lazy naming scheme. Then, it's just up to the people to use it or not.

**Reply by daipom** [reply]
I made some comments on the PR.
If you wait a week or so and there is no response, let's consider releasing the fork as a new gem!
